opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -1086,6 +1098,65 @@ def make_model_config_json(
                 "all_config": json.dumps(all_config),
             },
         }
+
+        if pooling_mode is not None:
```

Comment:
Let's add an unit test for this function.

Reply:
Added